docs/product/insights/web-vitals/index.mdx

```diff
@@ -110,3 +110,22 @@ For an interaction to be considered an INP candidate, it must have a latency of
 **Opportunity** scores are number values associated with each page and are meant to give you a sense of which pages are most valuable to improve. The Opportunity score is the maximum possible increase to your application's overall Performance Score if you were to raise a page's score to 100. If a page in your application already has a score of 100, the Opportunity score would be 0, since there's no way to further optimize this page.
 
 Opportunity score calculation is also weighted based on the traffic that a page sees. Pages with high traffic tend to have higher Opportunity scores. For example, a page with a score of 70 that experiences 1000 page loads a day will have a higher Opportunity score than a page with a score of 50 that experiences only 10 page loads a day.
+
+### Prerequisites and Limitations
+
+**Web Vitals** and **Performance Scores** are automatically captured by Sentry's JavaScript SDKs through the [BrowserTracing](/platforms/javascript/configuration/integrations/browsertracing/) integration. 
+
+You can find out how to set up Sentry on your web application in the [installation docs](/platforms/javascript/). If you are using a JavaScript framework, you may need to follow additional setup instructions for your specific framework.
+
+- [React SDK](/platforms/javascript/guides/react/)
+- [Vue SDK](/platforms/javascript/guides/vue/)
+- [Angular SDK](/platforms/javascript/guides/angular/)
+- [Svelte SDK](/platforms/javascript/guides/svelte/)
+
+Although Sentry provides auto instrumentation for many frontend frameworks, there are some environments and scenarios that may not be fully supported. If you are unable to find your pages in the **Web Vitals** page, check to see if your page loads are meeting the following conditions:
+
+- **Performance Scores** do not support mobile browsers.
+- **Performance Scores** only support **Chrome**, **Firefox**, **Safari**, **Opera**, and **Edge** desktop browsers. 
+- A page load must capture a minimum set of web vitals in order to be eligible for **Performance Score** calculation. e.g. Chrome page loads must contain LCP, FCP, CLS, and TTFB, otherwise they will not appear in the Web Vitals page.
```

Comment:
```suggestion
- A page load must capture a minimum set of web vitals in order to be eligible for **Performance Score** calculation. For example, Chrome page loads must contain LCP, FCP, CLS, and TTFB; otherwise they will not appear in the Web Vitals page.
```
We try to avoid latin abbreviations for non-native speakers